nilearn/_utils/estimator_checks.py

```diff
@@ -1603,6 +1606,137 @@ def check_img_estimator_n_elements(estimator):
                 getattr(estimator, method)(X)
 
 
+def check_img_estimator_standardization(estimator_orig):
+    """Check estimator with several value for standardize.
+
+    Check that the different values give different results.
+
+    For _BaseDecomposition derived classes
+    this is mostly only a smoke test.
+    """
```

Comment:
This test is pretty long. Would it make sense to split it ?

Reply:
will try to split it

Reply:
note that splitting the test leads to a bit of duplication